exercises/Second largest number/second_largest_num.py

```diff
@@ -0,0 +1,15 @@
+
+def second():
+    with open("input.txt", "r") as f:
+        line = f.readlines()
+    num = line[0].rstrip()
+    maximum = max(num)
+    with open("output.txt", "w+") as o:
+        if len(num) == 1 or len(set(num.split(" "))) == 1:
```

Comment:
No need for using `set(...)` here. You can just check if ´x for x in num if x != maximum´ is empty.